### ISSUE TITLE
Question on Online Function in Custom Variables Section

### ISSUE THREAD

**Issue by aaronhatesregex** [gpakosz/.tmux#451]
TL;DR:

1. What is the meaning you intend to tell by `ping -t 1`(*time to live* or *timeout* or something else)?
2. It seems it's not portable.

Long story:

### Preface

I setup .tmuxconf.local long time ago and renewed it recently. The issue is I always got a "✘" even when I am of course online, using the `online` function provided in `custom variables` section. I'm sure I have access to CloudFlare DNS. So I dig it up.

The command is

    ping -t 1 -c 1 1.0.0.1 >/dev/null 2>&1 && printf '✔' || printf '✘'`

> Maybe the timeout is too low in which case you can try -t 2 or more?

I assume by `-t` you mean *timeout* in the quote from #444.

### Ping

Actually, there are `iputils-ping` and `inetutils-ping` and according to [Unix & Linux Stack Exchange](https://unix.stackexchange.com/questions/400351/what-are-the-differences-between-iputils-ping-and-inetutils-ping), `iputils-ping` is Linux-specific while `inetutils-ping` is the portable GNU implementation.

`iputils-ping` is installed on my Debian testing machine. From the man page (for an online version, see [here](https://manpages.debian.org/testing/iputils-ping/ping.8.en.html) but not [here](https://manpages.debian.org/testing/inetutils-ping/ping.1.en.html)), `-t` means *ttl* and `-W` means *timeout*. So my issue is solved by replacing `-t` with `-W` but it's not portable. However, [GNU manpage](https://www.gnu.org/software/inetutils/manual/html_node/Ping-options.html#Ping-options) tells me that `-w` means *timeout* and `-t` means *type* and

> -t type
> --type=type
>     Send type packets. Accepted values are ‘address’, ‘echo’, ‘mask’, and ‘timestamp’.

Now I'm kinda confused by the `-t 1` in online function XD.

### Portability

And a sad fact is that `iputils-ping` does not support long option, so I cannot make it portable by using `--timeout=1`.

**Comment by gpakosz** [comment]
Hello @aaronhatesregex 👋

Indeed with `-t 1` I meant timeout.
I knew `ping` was not so portable that's why I made it an example of a local function.
Then I forgot `-t` was not portable at least between macOS and Linux.

I will just remove `-t 1` in the sample `.tmux.conf.local` file.

Would that work for you?

**Comment by aaronhatesregex** [comment]
> I will just remove -t 1 in the sample .tmux.conf.local file.
> Would that work for you?

Actually I don't use it myself, just wondering why it doesn't work. So, sure and maybe you can add a notice to prevent people (like me :facepalm: )  from uncommenting the function.

At first I think I should remove the comment to make the function work, but I got a `unknown command`. Then I see `cut` to remove the first three letters and know I should keep them.

**Comment by gpakosz** [comment]
> maybe you can add a notice to prevent people (like me 🤦 ) from uncommenting the function.

Ah yeah the function looks commented because `.tmux.conf.local` is a polyglot
